source/openapi-admin-v3.yaml

```diff
@@ -1989,6 +1989,41 @@ paths:
       responses:
         "404":
           description: No IP addresses or CIDR blocks to delete
+  "/groups/{groupId}/apps/{appId}/metrics":
+    get:
+      tags:
+        - metrics
+      operationId: adminGetMetrics
+      summary: Retrieve App Services metrics
```

Comment:
Maybe also add a `description` field with more details/links out? e.g. what metrics are available, how long do we hang on to metrics data, etc.